src/lib/logger.ts

```diff
@@ -55,6 +56,24 @@ const browserLoggerFn: LoggerFunction = (level, data) => {
   )
 }
 
+/**
+ * For use in the mobile app where colorized logging isn't supported
+ */
+const simpleLoggerFn: LoggerFunction = (level, data) => {
+  const consoleFn = level === LogLevel.TRACE ? LogLevel.DEBUG : level
+
+  const msg = data.error
+    ? `Error [Kind: ${data.error.kind}] ${data.error.message} `
+    : data._messages?.join(' ') || data.message
+
+  // eslint-disable-next-line no-console
+  console[consoleFn](
+    `${logStyles[level].icon} ${level}: [${data.namespace}] ${msg}\n`,
+    // In the webview, objects are rendered as "[object Object]" so we'll format it
+    require('util').inspect(data),
```

Comment:
as discussed when @cjthompson confirms this works in iOS (we're guessing it's a polyfill) i'm happy to approve this.

verified logging is working fine in ephemeral environment.

Reply:
confirmed working in iOS per @cjthompson